infra/base-images/base-builder/Dockerfile

```diff
@@ -123,14 +123,23 @@ RUN cd $SRC && \
     tar -xzv --strip-components=1 -f $SRC/oss-fuzz.tar.gz && \
     rm -rf examples $SRC/oss-fuzz.tar.gz
 
+RUN cd $SRC && \
+    git clone --depth 1 https://github.com/google/centipede.git && \
+    echo 'build --cxxopt=-stdlib=libc++ --linkopt=-lc++' \
```

Comment:
Move this closer to the point where it's used (i.e. in the `precompile_centipede` file).

Reply:
Done : )